frontend/src/lib-customizations/espoo/employee-mobile-frontend/assets/i18n/fi.ts

```diff
@@ -238,7 +238,9 @@ export const fi = {
       differenceReason: 'Kirjaa syy',
       differenceInfo:
         'Saavut työvuorostasi poikkeavaan aikaan. Valitse syykoodi.',
-      differenceInfoToggle: 'Poikkeaa työvuorostasi'
+      differenceInfoToggle: 'Poikkeaa työvuorostasi',
+      hasFutureAttendance:
+        'Sinulla on tulevaisuuteen merkattu läsnäolo, joten et voi kirjautua läsnäolevaksi.'
```

Comment:
`merkattu` is slang, `merkitty` would be the correct word.

Source: https://fi.wiktionary.org/wiki/merkata